pyproject.toml

```diff
@@ -75,6 +75,8 @@ filelock = "*"
 ruamel-yaml = "^0.18.5"
 pymatreader = "^0.0.32"
 xmltodict = "*"
+matplotlib = "*"
```

Comment:
@milesAraya 

Once commented, but first, I think it would be good if you could share the following points with us.

- Test Patterns
  - Are there any missing points to be checked?
  - Processing performance is also a target to be checked (cases where a large number of images are processed are also assumed)
- Test results of the above patterns
  - Comparison of kaleido/matplotlib processing results, etc.

For the above test, it may be necessary to secure an environment in which kaleido works (e.g. Mac M2?).